Exercise_1/problem-2.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"time"
+)
+
+var set int = 0 //used to switch between go routines
+var count int = 0
```

Comment:
Can you try for a solution which doesn't use global variables? Global variables are usually avoided unless absolutely necessary.